rpc/cc/arduino/cli/commands/v1/common.proto

```diff
@@ -141,6 +144,9 @@ message PlatformRelease {
   bool missing_metadata = 7;
   // True this release is deprecated
   bool deprecated = 8;
+  // True if the platform dependencies are available for the current OS. This
+  // also means that the platform is installable.
```

Comment:
Besides the OS, please mention the arch in the public API. It's nicely documented in the code. Thanks

> // true if at all ToolDependencies are available for the current OS/ARCH.